HealthVault/Classes/Utilities/MHVValidator.h

```diff
@@ -158,5 +158,6 @@ while(NO)
                                                 return hr; \
                                             } \
 
-MHVClientResult* MHVValidateCollection(MHVCollection *collection, MHVClientResultCode error);
+MHVClientResult* MHVValidateCollection(MHVCollectionInternal *collection, MHVClientResultCode error);
```

Comment:
MHVValidateCollection will need to be removed once MHVCollectionInternal is deleted.